packages/nodes-base/nodes/Jira/IssueDescription.ts

```diff
@@ -20,41 +20,49 @@ export const issueOperations: INodeProperties[] = [
 				name: 'Changelog',
 				value: 'changelog',
 				description: 'Get issue changelog',
+				action: 'Get an issue changelog',
 			},
 			{
 				name: 'Create',
 				value: 'create',
 				description: 'Create a new issue',
+				action: 'Create an issue',
 			},
 			{
 				name: 'Delete',
 				value: 'delete',
 				description: 'Delete an issue',
+				action: 'Delete an issue',
 			},
 			{
 				name: 'Get',
 				value: 'get',
 				description: 'Get an issue',
+				action: 'Get an issue',
 			},
 			{
 				name: 'Get All',
 				value: 'getAll',
 				description: 'Get all issues',
+				action: 'Get all issues',
 			},
 			{
 				name: 'Notify',
 				value: 'notify',
 				description: 'Create an email notification for an issue and add it to the mail queue',
+				action: 'Notify an issue',
```

Comment:
Not sure what it should be but it doesn't look right. Maybe use "Create an email notification for an issue"